wordpress-mobile/WordPress-Android
# Add Stats widget tracking

This PR adds tracking events for adding and removing of the widgets. I've renamed the enum `ViewType` to `WidgetType` because it's widget specific.

To test:
* Check that the events are being sent when a widget is added/removed

Update release notes:
